Lib/ufo2ft/featureWriters/markFeatureWriter.py

```diff
@@ -326,6 +326,10 @@ def _getAnchorLists(self):
                         "unnamed anchor discarded in glyph '%s'", glyphName
                     )
                     continue
+                # Skip anchors not meant for export
+                # See https://github.com/googlefonts/ufo2ft/issues/477
+                if anchorName.startswith(("#", "_#")):
```

Comment:
This should check for anchors started by any non-letter character, not only `#`.